src/components/signature/signature.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import Signature from './signature';
+
+const testData = [
+  {
+    author: 'John Doe',
+    date: '2020-07-01',
+  },
+  {
+    author: 'John Doe',
+    date: new Date().toString(),
+  },
+  {
+    author: 'John Doe',
+    date: '2020-08-01',
+  },
+];
+
+describe(Signature.name, () => {
```

Comment:
Please use a static name I don't see the point of using some variable reference here. Makes it hard to find in the output.